packages/microsoft_defender_endpoint/data_stream/vulnerability/manifest.yml

```
@@ -0,0 +1,84 @@
+title: Collect Microsoft Defender for Endpoint vulnerability and affected machine logs from API
+type: logs
+ilm_policy: logs-microsoft_defender_endpoint.vulnerability-default_policy
+streams:
+  - input: cel
+    title: Microsoft Defender Endpoint Vulnerabilities
+    description: Collect Microsoft Defender for Endpoint vulnerability and affected machine logs from API.
+    enabled: false
+    template_path: cel.yml.hbs
+    vars:
+      - name: interval
+        type: text
+        title: Interval
+        description: Duration between requests to the Microsoft Defender Endpoint Vulnerability API. Supported units for this parameter are h/m/s.
+        multi: false
+        required: true
+        show_user: true
+        default: 4h
+      - name: batch_size
+        type: integer
+        title: Batch Size
+        description: Specifies how many records to return in a single request of the Microsoft Defender Endpoint Vulnerability API.
+        multi: false
+        required: true
+        show_user: false
+        default: 8000
+      - name: collect_only_vulnerability_with_affected_machine
```

```suggestion
      - name: affected_machines_only
```
The option name is too big, it need not be. The title and description are what users see.